src/custom/connectors/index.ts

```diff
@@ -1,16 +1,110 @@
-import { ChainId } from '@uniswap/sdk'
+import { Web3Provider } from '@ethersproject/providers'
 import { InjectedConnector } from '@web3-react/injected-connector'
+import { WalletConnectConnector } from '@web3-react/walletconnect-connector'
+import { WalletLinkConnector } from '@web3-react/walletlink-connector'
+import { PortisConnector } from '@web3-react/portis-connector'
 
-export {
-  NETWORK_CHAIN_ID,
-  fortmatic,
-  portis,
-  network,
-  walletconnect,
-  walletlink,
-  getNetworkLibrary
-} from '@src/connectors'
-
-export const injected = new InjectedConnector({
-  supportedChainIds: [ChainId.MAINNET, ChainId.ROPSTEN, ChainId.RINKEBY, ChainId.GÖRLI, ChainId.KOVAN, ChainId.XDAI]
+import { FortmaticConnector } from 'connectors/Fortmatic'
+import { NetworkConnector } from 'connectors/NetworkConnector'
+
+const SUPPORTED_CHAIN_IDS = process.env.REACT_APP_SUPPORTED_CHAIN_IDS
+const DEFAULT_CHAIN_ID = Number(process.env.REACT_APP_CHAIN_ID || '1')
+
+const FORMATIC_KEY = process.env.REACT_APP_FORTMATIC_KEY
```

Comment:
should this and `L14` have defaults and/or consider `undefined` keys below?

Reply:
For now I wouldn't want to change too much this. 

This is mostly the logic of uniswap. I wouldn't want to hardcode it here initally. Probably if I change their behaviour would be to make it optional, and not include these providers if the env is not present. Anyways, it goes beyond the scope of this PR